actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -142,8 +142,20 @@ def add_static_dependency(dependencies, dependency)
           end
         end
 
+        def resolve_directories(wildcard_dependencies)
+          wildcard_dependencies.map do |dependency|
+            Dir[ "digestor/#{dependency}" ].map do |file|
```

Comment:
After having looked at this some more, we probably need to give the dependency tracker access to either a `view_paths` or a `lookup_context`.
I still don't know enough about either view paths or lookup contexts, to say if this can work out yet. I'll keep investigating.